.github/CODEOWNERS

```diff
@@ -493,7 +493,7 @@ pep-0635.rst  @brandtbucher @gvanrossum
 pep-0636.rst  @brandtbucher @gvanrossum
 pep-0637.rst  @stevendaprano
 pep-0638.rst  @markshannon
-pep-0639.rst  @pfmoore
+pep-0639.rst  @pfmoore @CAM-Gerlach
```

Comment:
On a pedantic reading of PEP 1, I don't think PEP editors are allowed to be listed in CODEOWNERS.

> Update .github/CODEOWNERS [7] such that any core developer co-author(s) or sponsor are listed for your new file such that any future pull requests will be assigned to them.

A PEP editor can be a sponsor, but the case of PEP editors being authors seems to be missed as an edge case. 

However, I think it makes sense, so probably better to update PEP 1 to allow this? If other @python/pep-editors  agree I could propose an update the affected lines.

A

Reply:
Good catch. I'm already listed as the owner of PEPs 673 and 675 and I'm not a core dev, so in practice we're not following this rule already. Let's amend PEP 1 to make it clear. I'd suggest wording like "any author or sponsor with write access to the PEPs repository".

Reply:
See #2252 

A

Reply:
Yeah, that certainly seems like a clear oversight to me; PEP 1 may not have envisioned PEP editors who weren't core developers, like you, me and @JelleZijlstra are today—and it affects you too on your PEP 676, since you're the author but not a sponsor either.